src/main/java/edu/wpi/grip/ui/Main.java

```diff
@@ -31,8 +31,7 @@ public void start(Stage stage) {
          * Any exceptions thrown by the UI will be caught here and an exception dialog will be displayed
          */
         Thread.currentThread().setUncaughtExceptionHandler((thread, throwable) -> {
-            this.eventBus.post(new FatalErrorEvent(throwable));
-
+            this.eventBus.post(new UnexpectedThrowableEvent(throwable));
```

Comment:
Why?
JavaFX hits null pointers all the time. I'm working on fixing that. Some of these things can be easily resolved by simply closing and reopening a preview for example.